### ISSUE TITLE
Add an option to provide a custom layout/format

### ISSUE THREAD

**Issue by treehead** [alct/export-tabs-urls#7]
I would like the ability to export a list of titles and URLs in Markdown format, which look like this example:

```markdown
- [Mozilla](https://www.mozilla.org/)
- [MDN Web Docs](https://developer.mozilla.org/)
- [GitHub](https://github.com/)
```

I realize that this may not be everyone's preferred output, and that others may wish to export in other formats, so perhaps the enhancement could be to add one or more custom output formats?

**Comment by alct** [comment]
Hello,

ETU already allows for different formats, the "Include titles" option is just another format (see https://github.com/alct/export-tabs-urls/blob/master/popup/popup.js#L82 and https://github.com/alct/export-tabs-urls/blob/master/popup/popup.js#L86 ). This shouldn't be hard to implement and could in its "custom format" rather than its "markdown format" version, indeed, be useful.

Note that I want to keep ETU clean and straightforward, so adding yet another checkbox or a field on the main popup isn't an option. Right now, the best UI/X that I can think of would be to:

- have a dedicated page/whatever for options (such as "Custom format" or "Ignore non-HTTP(S) URLs",...);

- if a custom format is provided, replace the "Include titles" checkbox on the main popup by a "Custom format" one.

If you have any other idea, I would be grateful to hear your suggestions on the subject.

This is directly dependent on #1 (which will implement the option storage mechanism) and is of lesser priority than #6.

**Comment by alct** [comment]
Implemented in 0.2.0, see 5239d04218942ae7f7539b4b1b541fa1311d3aa9
